nilearn/datasets/atlas.py

```diff
@@ -20,6 +20,112 @@
 _TALAIRACH_LEVELS = ['hemisphere', 'lobe', 'gyrus', 'tissue', 'ba']
 
 
+def fetch_difumo(dimension=64, resolution_mm=2, data_dir=None, resume=True, verbose=1):
```

Comment:
Sorry for getting to this a bit late. All the atlas fetchers have names that start with ```fetch_atlas_*```. May be ```fetch_atlas_difumo```.